changelog/unreleased/change-replace-deprecated-substr

```diff
@@ -0,0 +1,7 @@
+Change: Replace deprecated String.prototype.substr()
```

Comment:
```suggestion
Enhancement: Replace deprecated String.prototype.substr()
```

Since this isn't a user-facing breaking change it counts as `enhancement` in SemVer, could you update the changelog item filename & heading?:)

Reply:
Sure. I updated the file as requested